src/command.rs

```diff
@@ -185,14 +198,20 @@ impl CommandExt for Command {
             let err = TopgradeError::ProcessFailedWithOutput(program, output.status, stderr.into_owned());
 
             let ret = Err(err).with_context(|| message);
-            debug!("Command failed: {ret:?}");
+            debug!(
+                "{}",
+                t!(
+                    "Command failed: {error_and_context}",
```

Comment:
That's a good question 🤔 On one side they are syntactically the same on the other they differ semantically... I think it would be easier to consider the same keys but with different placeholders as duplicates

Reply:
> I think it would be easier to consider the same keys but with different placeholders as duplicates

Yeah, this is easier to implement since they are literally different 😁